tests/bluetooth/audio/cap_commander/src/test_broadcast_reception.c

```diff
@@ -0,0 +1,372 @@
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+}
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_pa_interval_low)
+{
+	int err;
+
+	fixture->start_param.param[0].pa_interval = BT_GAP_PER_ADV_MIN_INTERVAL - 1;
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+}
+
+/*
+ * Test for pa_interval_high omitted
+ * pa_interval is a uint16_t, BT_GAP_PER_ADV_MAX_INTERVAL is defined as 0xFFFF
+ * and therefor we can not test in the current implementation
+ */
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_invalid_broadcast_id)
+{
+	int err;
+
+	fixture->start_param.param[0].broadcast_id = BT_AUDIO_BROADCAST_ID_MAX + 1;
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+}
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_no_subgroups)
+{
+	int err;
+
+	fixture->start_param.param[0].num_subgroups = 0;
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+}
+
+ZTEST_F(cap_commander_test_broadcast_reception,
+	test_commander_reception_start_invalid_num_subgroups)
+{
+	int err;
+
+	fixture->start_param.param[0].num_subgroups = CONFIG_BT_BAP_BASS_MAX_SUBGROUPS + 1;
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+}
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_duplicate_bis_sync)
+{
+#if CONFIG_BT_BAP_BASS_MAX_SUBGROUPS > 1
+	int err;
+
+	fixture->start_param.param[0].subgroups[0].bis_sync =
+		fixture->start_param.param[0].subgroups[1].bis_sync;
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+#else
+	ztest_test_skip();
+#endif
```

Comment:
Everywhere else the Z_TEST_SKIP_IFNDEF is only used for checking if a CONFIG is set or not. However I modified the code so that it is consistent with what is done in `tests\bluetooth\audio\ascs\src\main.c`

Reply:
> Z_TEST_SKIP_IFNDEF is only used for checking if a CONFIG

That's clearly also what it was defined for, but looking at the implemenation it's a simply `COND_CODE_1` :)